content/c-sharp/concepts/methods/methods.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: 'Methods' 
+Description: 'Methods in C# are a block of code that can be called.'
+Subjects: 
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+CatalogContent: 
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+A **method** is a block of code used in [C#](https://www.codecademy.com/resources/docs/c-sharp) (and many other programming languages) containing statements. Each C# application has at least one method, called the 'Main', which is the starting point for the application. Methods are used to avoid the copy pasting of code, enabling better maintainability and readability of the code. Usually, a method has one or multiple dedicated purposes. 
```

Comment:
```suggestion
A **method** is a block of code used in C# (and many other programming languages) containing statements. Each C# application has at least one method, called `Main`, which is the starting point for the application. Methods are used to avoid the copy pasting of code, enabling better maintainability and readability of the code. Usually, a method has one or multiple dedicated purposes. 
```

Reply:
We generally don't link back to the parent entry (The navigation in Docs already provides those links in the top of the entry)